test/unit/modules/blocks/utils.js

```diff
@@ -13,268 +13,558 @@
+				(err, data) => {
+					expect(err).to.equal('Address error stub');
+					expect(data).to.be.undefined;
+					done();
+				}
 			);
+		});
 
-			it('should set this.applied to 0');
-
-			describe('reset function', () => {
-				it('should set this.applied to 0');
-			});
-
-			describe('applyNext function', () => {
-				describe('when this.applied >= this.target', () => {
-					it('should throw error');
-				});
-
-				it('should increment this.applied');
-
-				describe('when this.applied = 1', () => {
-					it('should call this.log');
-				});
-
-				describe('when this.applied = this.target', () => {
-					it('should call this.log');
-				});
-
-				describe('when this.applied %this.step = 1', () => {
-					it('should call this.log');
-				});
-			});
+		it('should return error when account not found', done => {
+			blocksUtilsModule.aggregateBlocksReward(
+				{ address: '0L' },
+				(err, data) => {
+					expect(err).to.equal('Account not found');
+					expect(data).to.be.undefined;
+					done();
+				}
+			);
+		});
 
-			describe('log function', () => {
-				it('should call library.logger.info with msg');
+		it('should return error when library.db.blocks.aggregateBlocksReward fails', done => {
+			blocksUtilsModule.aggregateBlocksReward(
+				{ address: '1L' },
+				(err, data) => {
+					expect(loggerStub.error.args[0][0]).to.contains(
+						"TypeError: Cannot read property '0' of undefined"
+					);
+					expect(err).to.equal('Blocks#aggregateBlocksReward error');
+					expect(data).to.be.undefined;
+					done();
+				}
+			);
+		});
 
-				it('should call library.logger.info with message');
-			});
+		it('should return error when account is not a delegate', done => {
+			library.db.blocks.aggregateBlocksReward = sinonSandbox
+				.stub()
+				.resolves([{ delegate: null }]);
+
+			blocksUtilsModule.aggregateBlocksReward(
+				{ address: '1L' },
+				(err, data) => {
+					expect(err).to.equal('Account is not a delegate');
+					expect(data).to.be.undefined;
+					done();
+				}
+			);
 		});
 
-		it('should return a new instance of BlockProgressLogger');
+		it('should return block object', done => {
+			library.db.blocks.aggregateBlocksReward = sinonSandbox
+				.stub()
+				.resolves([{ delegate: '123abc', fees: 1, count: 100 }]);
+
+			blocksUtilsModule.aggregateBlocksReward(
+				{ address: '1L' },
+				(err, data) => {
+					expect(data).to.be.an('object');
+					expect(data.fees).to.equal(1);
+					expect(data.count).to.equal(100);
+					expect(data.rewards).to.equal('0');
+					done();
+				}
+			);
+		});
 	});
 
 	describe('onBind', () => {
-		it(
-			'should call library.logger.trace with "Blocks->Utils: Shared modules bind."'
-		);
+		beforeEach(() => {
+			loggerStub.trace.reset();
+			blocksUtilsModule.onBind(modulesStub);
+		});
+
+		it('should call library.logger.trace with "Blocks->Utils: Shared modules bind."', () => {
+			expect(loggerStub.trace.args[0][0]).to.equal(
+				'Blocks->Utils: Shared modules bind.'
+			);
```

Comment:
I believe the feedback https://github.com/LiskHQ/lisk/pull/1458#discussion_r166598190 is missed out. 

Even if we use `sinon-chai` the assertion would be: 

```
exppect(loggerStub.trace).to.have.been.calledOnce;
exppect(loggerStub.trace).to.have.been.calledWith('Blocks->Utils: Shared modules bind.');
```